### ISSUE TITLE
Swin-T 224x224 QAT: not able to reproduce 81.00% accuracy

### ISSUE THREAD

**Issue by gcunhase** [NVIDIA/FasterTransformer#685]
### Branch/Tag/Commit

main

### Docker Image Version

nvcr.io/nvidia/pytorch:22.09-py3

### GPU name

RTX3090, DGX1V

### CUDA Driver

470.182.03

#### Issue
Not able to reproduce the [expected accuracy](https://github.com/NVIDIA/FasterTransformer/tree/main/examples/pytorch/swin/Swin-Transformer-Quantization#difference-between---int8-mode-1-and---int8-mode-2):

| Mode | Accuracy-1 (%) |
| ---- | -------------- |
| **PTQ** (expected) | 80.75 |
| PTQ (obtained) | 80.70 (-0.05) |
| **QAT** (expected) | 81.00 |
| QAT (obtained) | 80.83 (-0.17) |

#### Reproduce
Followed the steps [here](https://github.com/NVIDIA/FasterTransformer/tree/main/examples/pytorch/swin/Swin-Transformer-Quantization):
1. Calibrate Swin-T:

```shell
python -m torch.distributed.launch --nproc_per_node 1 \
  --master_port 12345 main.py \
  --calib \
  --cfg SwinTransformer/configs/swin_tiny_patch4_window7_224.yaml \
  --resume swin_tiny_patch4_window7_224.pth \
  --data-path <imagenet-path> \ 
  --num-calib-batch 10 \
  --calib-batchsz 8 \
  --int8-mode 1 \
  --calib-output-path calib-checkpoint
```

2. Perform QAT:
```shell
python -m torch.distributed.launch --nproc_per_node 4 \
    --master_port 12345 main.py \
    --train \
    --cfg SwinTransformer/configs/swin_tiny_patch4_window7_224.yaml \
    --resume ./calib-checkpoint/swin_tiny_patch4_window7_224_calib.pth \
    --data-path /data/datasets/ILSVRC2012 \
    --quant-mode ft2 \
    --teacher swin_tiny_patch4_window7_224.pth \
    --output qat-output \
    --distill \
    --int8-mode 1 \
    --batch-size 128 \
    --num-epochs 5 \
    --qat-lr 1e-5
```

#### Question
What hyper-parameters did you use to obtain the reported accuracies for both PTQ and QAT? Thanks.

**Comment by gcunhase** [comment]
Running the QAT command in [qat.sh](https://github.com/NVIDIA/FasterTransformer/blob/main/examples/pytorch/swin/Swin-Transformer-Quantization/qat.sh) resulted in an accuracy of 80.95%, which is closer to the reported 81.00%. Would you consider this 0.05 point drop acceptable?
